backend/docs/api/management_v2_deployments.yaml

```
@@ -571,6 +571,105 @@ paths:
         Lists all release update types.
       tags:
       - Management API
+  /artifacts/list:
+    get:
+      description: |
+        Returns a collection of all artifacts.
+      operationId: List Artifacts with pagination
+      parameters:
+      - description: | 
+          Artifact(s) exact name(s) or prefix filter. 
+          Add `*` to the end for prefix matching.
+        in: query
+        name: name
+        schema:
+          type: array
+          items:
+            type: string
+        style: form
+        explode: true
+        examples:
+          Single Name:
+            value:
+              - foo
+          Multiple Names:
+            value:
+              - foo
+              - bar
+          Prefix Search:
+            value:
+              - foo*
```

A summary would be nice :)
```suggestion
          single:
            summary: Get release with the given name.
            value:
              - foo
          multi:
            summary: Get releases matching multiple names by repeating the query parameter.
            value:
              - foo
              - bar
          prefix:
            summary: Get releases with the specified prefix
            value:
              - foo*
```